network-api/networkapi/mozfest/templates/partials/primary_mozfest_nav.html

```diff
@@ -11,8 +11,8 @@
               <div class="row">
                 <div class="col">
                   <div class="nav-links pt-3">
-                    <div><a class="{% primary_active_nav request '/' %}" href="/">{{ root_name }}</a></div>
-                    {% include "../fragments/nav-links.html" with pre="<div>" post="</div>" %}
+                    <div><a class="{% primary_active_nav request menu_root.full_url menu_root.full_url %}" href="{{ menu_root.url }}">{{ root_name }}</a></div>
```

Comment:
`.full_url` is used for code comparison purposes because `.url` is not stable (it might or might not yield a relative or absolute path).

However, for the actual `href` attribute it doesn't matter whether the `.url` value is relative or absolute. All that matters is that it's correct in terms of browser navigation.